app/parsers/bulkrax/bagit_parser.rb

```diff
@@ -3,6 +3,7 @@
 module Bulkrax
   class BagitParser < CsvParser # rubocop:disable Metrics/ClassLength
     include ExportBehavior
+    require 'bagit'
```

Comment:
Does this have to be required within the `BagitParser` class? Every other `require` I see in the app are listed at the top of the file outside of the class